pandas/tests/test_window.py

```diff
@@ -159,33 +157,19 @@ def compare(result, expected):
         expected = pd.concat([a_mean, a_sum, b_mean, b_sum], axis=1)
         expected.columns = pd.MultiIndex.from_tuples([('A', 'mean'), (
             'A', 'sum'), ('B', 'mean2'), ('B', 'sum2')])
-        compare(result, expected)
+        assert_frame_equal(result, expected, check_like=True)
 
         result = r.aggregate({'A': ['mean', 'std'], 'B': ['mean', 'std']})
         expected = pd.concat([a_mean, a_std, b_mean, b_std], axis=1)
         expected.columns = pd.MultiIndex.from_tuples([('A', 'mean'), (
             'A', 'std'), ('B', 'mean'), ('B', 'std')])
-        compare(result, expected)
-
-        result = r.aggregate({'r1': {'A': ['mean', 'sum']},
-                              'r2': {'B': ['mean', 'sum']}})
-        expected = pd.concat([a_mean, a_sum, b_mean, b_sum], axis=1)
-        expected.columns = pd.MultiIndex.from_tuples([('r1', 'A', 'mean'), (
-            'r1', 'A', 'sum'), ('r2', 'B', 'mean'), ('r2', 'B', 'sum')])
-        compare(result, expected)
-
-        result = r.agg({'A': {'ra': ['mean', 'std']},
-                        'B': {'rb': ['mean', 'std']}})
```

Comment:
just moved, these are now in the 'nested_dict' tests (both the first part which now raises and the 2nd part which is ok)